docs/source/index.md

```diff
@@ -106,140 +147,55 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
 | **Continuous** | online purchases         | ad conversion time      |
 | **Discrete**   | concerts & sports events | recurring subscriptions |
 
-## Installation
-
-PyMC-Marketing requires **Python 3.10 or greater**.
-
-Install and activate an environment (e.g. `marketing_env`) with the `pymc-marketing` package from [conda-forge](https://conda-forge.org). It may look something like the following:
-
-```bash
-conda create -c conda-forge -n marketing_env pymc-marketing
-conda activate marketing_env
-```
-
-### Installation for developers
-If you are a developer of pymc-marketing, or want to start contributing, [refer to the contributing guide](https://github.com/pymc-labs/pymc-marketing/blob/main/CONTRIBUTING.md) to get started.
-
-See the official [PyMC installation guide](https://www.pymc.io/projects/docs/en/latest/installation.html) if more detail is needed.
-
-## Quickstart
-
-Create a new Jupyter notebook with either JupyterLab or VS Code.
 
-### JupyterLab Notebook
 
-After installing the `pymc-marketing` package (see above), run the following with `marketing_env` activated:
+## Resources
```

Comment:
Can this be centered? There is quite a bit of content listed here and all the "sections" sort of run together right now.

Reply:
Fixed in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/926f5b545252dc64f0f15a75617390401787d98d